install/0000_00_cluster-version-operator_03_deployment.yaml

```diff
@@ -50,7 +50,17 @@ spec:
         node-role.kubernetes.io/master: ""
       priorityClassName: "system-cluster-critical"
       tolerations:
-        - operator: Exists
+      - key: node-role.kubernetes.io/master  
+        operator: Exists
+        effect: "NoSchedule"
+      - key: "node.kubernetes.io/unreachable"
+        operator: "Exists"
+        effect: "NoExecute"
+        tolerationSeconds: 120 
+      - key: "node.kubernetes.io/not-ready"
```

Comment:
CVO must be scheduled on master so that it can drive installation even when it is `NotReady` ie not schedulable because it creates operators that create network and apiserver.

Reply:
To be clear, are you telling we should tolerate both `unschedulable` and `NotReady` taints?

Reply:
I want to say yes. ;)

We want to make sure CVO gets scheduled even when node is not ready because networking isn't setup yet.

Reply:
The cvo must always get a chance to land on any master.

Reply:
Including `memory` and `disk` pressures? I assume not..

Reply:
Updated with `unschedulable` and `network-unavailable` tolerations

Reply:
whay about not-ready NoSchedule ?